lib/resolver/javascript-universal.js

```diff
@@ -14,7 +14,11 @@ export default function ({ path, target }) {
   const isBuildIn = builtins.indexOf(target) > -1;
 
   if (isBuildIn) {
-    return `https://nodejs.org/api/${target}.html`;
+    let filename = target;
+    if (filename === 'module') {
+      filename = 'modules';
+    }
```

Comment:
What do you think about wrapping the package `builtins` in a separate file and extend the list there? By that we could easily reuse the list somewhere else, if needed.